### PR TITLE
net-vpn/i2p: version bump to 0.9.30, remove 0.9.28.

### PR DESCRIPTION
**Pull request by biergaizi**
Signed-off-by: Yifeng Li <tomli@tomli.me>

### CODE REVIEW

**Comment by monsieurp** [comment]
Could you please split the bump and the removal in two distinct commits ? Thanks!

**Comment by biergaizi** [comment]
Patch updated.
@monsieurp

**Comment by tharvik** [comment]
I guess we can remove the ecdsa flag then (thus forcing it).

**Comment by biergaizi** [comment]
ECDSA flag is removed.
@tharvik

**Comment by mgorny** [comment]
Hmm, something seems odd with highlights. @gentoo/java? @gentoo/Java?

File: net-vpn/i2p/i2p-0.9.30.ebuild [hunk 1]
```diff
@@ -31,17 +31,12 @@ DEPEND="${CP_DEPEND}
...
-			dev-java/oracle-jdk-bin
-		)
-	)
-	!ecdsa? ( >=virtual/jre-1.7 )"
+	dev-java/icedtea:7[-sunec]
+	dev-java/icedtea:8[-sunec]
+	dev-java/icedtea-bin:7
+	dev-java/icedtea-bin:8
+	dev-java/oracle-jre-bin
+	dev-java/oracle-jdk-bin"
```
Comment (tharvik) [inline net-vpn/i2p/i2p-0.9.30.ebuild]: It should still be an `||`, we do not need each version.

Comment (biergaizi) [inline net-vpn/i2p/i2p-0.9.30.ebuild]: Oops...A trap for new players 😃

**Comment by biergaizi** [comment]
Fixed.
@tharvik

**Comment by biergaizi** [comment]
What's wrong with the QA check?

**Comment by mgorny** [comment]
It's unhappy about the state of Gentoo. Nothing affecting your package, it seems.

**Comment by tharvik** [comment]
@chewi, LGTM.

**Pull request closed without merging**